### PR TITLE
Adds some new alt titles

### PR DESCRIPTION
**Pull request by SarmentiCampbell**
## About The Pull Request
Added more titles for jobs.

## Why It's Good For The Game
I mean, giving players a few more titles for their jobs won't hurt, right?

## Changelog
:cl:
add: Added new titles for jobs
/:cl:

### CODE REVIEW

**Comment by ShadeAware** [comment]
Ooo...
I -really- like this.

**Comment by ShadeAware** [comment]
Minor suggestion, remove the "worker" part of "Off-Duty Worker"

**Comment by NopemanMcHalt** [comment]
I like it! Main gripe, however, is when alternative titles used for command roles. I don't mind it personally, but some get confused a bit and some get offended by getting called HoP when they are "Crew Resources Officer".
Another thing is that Head of Staff is usually directed at Command roles in general - I don't think it could be used for HoP alt title. Good usage of Vice-Captain though! ~~even though it is purely sports ranks, I think~~

File: modular_skyrat/code/modules/jobs/job_types/_job_alt_titles.dm [hunk 1]
```diff
@@ -1,46 +1,90 @@
+//Command
+/datum/job/captain
+	alt_titles = list("Overseer", "Station Manager")
+
+/datum/job/hop
+	alt_titles = list("Crew Resources Officer", "Vice-Captain", "Head of Staff")
```
Comment (Akrilla) [inline modular_skyrat/code/modules/jobs/job_types/_job_alt_titles.dm:6]: Crew Resources Officer sounds a *little* too close to QM I think

**Pull request merged**